docs/api/categories.md

```diff
@@ -0,0 +1,265 @@
+    "categories": ["category_1_1", "category_1_2"],
+    "speechChallenges": [],
+    "progress": {
+      "done": 1,
+      "total": 2
+    }
+  },
+  {
+    "id": "category_2",
+    "created": "2017-01-12T09:36:20Z",
+    "updated": "2017-01-12T09:36:20Z",
+    "name": "Category 2",
+    "description": "Another awesome description.",
+    "color": "#0f0",
+    "imageUrl": "https://api.itslanguage.nl/download/UKbsMpBsXaJUsBbK",
+    "iconUrl": "https://api.itslanguage.nl/download/GdExSbs-ZVNnQUUe",
+    "categories": ["category_2_1", "category_2_2", "category_2_3"],
+    "speechChallenges": [],
+    "progress": {
+      "done": 1,
+      "total": 3
+    }
+  }
+]
+```
+
+# Get a single category
+
+### URL
+
+```http
+GET /category/:category HTTP/1.1
+```
+
+* `category` - **Required** The category identifier.
+
+### Request
+
+```http
+GET /category/category_1_1 HTTP/1.1
+Accept: application/json
+```
+
+### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "id": "category_1_1",
+  "created": "2017-01-12T09:38:20Z",
+  "updated": "2017-01-12T09:38:20Z",
+  "name": "Category 1.1",
+  "description": "Super duper.",
+  "color": "",
+  "imageUrl": "",
+  "iconUrl": "",
+  "categories": [],
+  "speechChallenges": [
+    {
+      "id": "speech_1",
+      "created": "2014-01-28T21:25:10Z",
+      "updated": "2014-01-28T21:25:10Z",
+      "topic": "What do you know about babies?",
+      "referenceAudioUrl": "https://api.itslanguage.nl/download/YsjdG37bUGseu8-bsJ",
+      "srtUrl": "https://api.itslanguage.nl/download/UKbsMpBsXaJUsBbK",
+      "imageUrl": "https://api.itslanguage.nl/download/GdExSbs-ZVNnQUUe"
+    },
+    {
+      "id": "12",
+      "created": "2014-01-28T21:25:10Z",
+      "updated": "2014-01-28T21:25:10Z",
+      "topic": "Book",
+      "referenceAudioUrl": "https://api.itslanguage.nl/download/YsjdG37bUGseu8-bsJ",
+      "srtUrl": "https://api.itslanguage.nl/download/UKbsMpBsXaJUsBbK",
+      "imageUrl": "https://api.itslanguage.nl/download/GdExSbs-ZVNnQUUe"
+    }
+  ],
+  "progress": {
+    "done": 1,
+    "total": 2
+  }
+}
+```
+
+### Create a catetory
+
+### URL
+
+```http
+POST /categories/category HTTP/1.1
+```
+
+### Request parameters
+
+Name               | Type      | Description
+-------------------|-----------|------------
+`id`               | `string`  | **Optional** The category identifier. If none is given, one is generated.
+`name`             | `string`  | **Optional** A name for the category.
+`description`      | `string`  | **Optional** A possible more verbose description about the category.
+`color`            | `string`  | **Optional** A color, preferably in RGB format.
+`image`            | `blob`    | **Optional** An image to show with the category.
+`icon`             | `blob`    | **Optional** An icon to show with the category.
+`categories`       | `object`  | **Optional** An array with category identifiers for nesting categories in a category.
```

Comment:
I think a subcategory can only be created when a parent category already exists. I don't see why a subcategory can be set in a category when creating the category. So in short, I think this param should be removed.